aries-site/src/examples/templates/feedback/PowerOnExample.js

```diff
@@ -0,0 +1,166 @@
+      // by Qualtrics in the response call which is what needs to be submitted
+      // back with the values to send the data.
+      values: {
+        fullURL: `url`,
+        deviceType: 'desktop',
+        QID1: value.value['power-rating-example'],
+        QID2_TEXT: value.value['power-textArea-example'],
+      },
+    };
+    // dev uses POST https://api.qualtrics.com/f1cad92018d2b-create-a-new-response
+    // call to send in the data collected.
+    // Contact Doris Singer for Survey ID, API Token, Response ID to use for testing
+    fetch(`https://api.qualtrics.com/f1cad92018d2b-create-a-new-response`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        Accept: 'application/json',
+        'X-API-TOKEN': 'Contact Doris Singer for API Token',
+      },
+      body: JSON.stringify(data),
+    })
+      .then(response => response.json())
+      .catch(error => {
+        console.error('Error:', error);
+      });
+  }, []);
+
+  return (
+    <Card width="large" gap="medium" justify="center" pad="medium">
+      <Box>
+        <Heading level={1} size="small" margin="none">
+          Power On
+        </Heading>
+        <Anchor href="#" label="Learn more about server actions" />
+      </Box>
+      {!isSuccessful ? (
+        <Text size="large">You are powering on 1 device</Text>
+      ) : (
+        <Box align="center" gap="xsmall" direction="row">
+          <StatusGoodSmall color="green" />
+          <Text size="large">Power On initiated for 1 device.</Text>
+        </Box>
+      )}
+      {!isLoading && !isSuccessful && (
+        <Text size="large">FTCUSAMountain_ML350Gen10_testcluster</Text>
+      )}
+      {isLoading && (
+        <Spinner
+          alignSelf="center"
+          size="medium"
+          message={{
+            start: 'Loading more data.',
+            end: 'Data has been loaded.',
+          }}
+        />
+      )}
+      {!isSuccessful && (
+        <Box
+          justify="end"
+          gap="medium"
+          margin={{ top: 'medium', bottom: 'small' }}
+          direction="row"
+        >
+          <Button label="Cancel" />
+          <Button
+            label="Power on 1 Device"
+            onClick={PowerOnLoading}
+            primary
+            type="submit"
+          />
+        </Box>
+      )}
+      {isSuccessful && (
+        <Box width="large" gap="medium" justify="center" pad="small">
+          <Heading level={4} size="small" margin="none">
+            Feedback on powering devices
+          </Heading>
+          <Form
+            method="post"
+            validate="submit"
+            kind="survey"
+            onChange={(nextValue, { touched }) => {
+              setRating(touched);
+            }}
+            onSubmit={value => {
+              console.log('submit', value.value);
+            }}
+          >
+            <Box gap="medium">
+              <FormField
```

Comment:
The spacing for this seems too large and does not match the instance at the top of the page.

![Screenshot 2023-01-06 at 4 00 52 PM](https://user-images.githubusercontent.com/1756948/211114324-cdd449fb-f992-4c18-9113-10c726d7ff33.png)


![Screenshot 2023-01-06 at 4 05 35 PM](https://user-images.githubusercontent.com/1756948/211114476-826488c7-0e56-42d7-bd33-0a025d51292f.png)

Looks like the Box around Heading and Form has gap we don't want. 

It also seems like we have some work to do in theme to tighten up the margins between labels and inputs. The gap between label and input should only be 6px. We have 12+12 = 24.

Reply:
https://github.com/grommet/grommet-theme-hpe/issues/299